src/docs/user/references/Notification-service.adoc

```diff
@@ -0,0 +1,257 @@
+- Activate one or more notification methods.
+
+==== Subscription types
+
+There are two types of <<_glossary_notification_subscription,*Subscriptions*>>:
+
+- Service subscriptions such as the Scheduler subscription to be notified when the <<_glossary_proactive_scheduler ,*Scheduler*>> changes its state
+or the Job planner subscription which can be activated with only one event type "All events"
+that corresponds to different issues the <<_glossary_job_planner,*Job Planner*>> may encounter.
+Check out the <<_job_planner_notifications,Job Planner Notifications>> to look at the full list of events.
+- Job subscriptions: To be notified of events that may occur during <<_glossary_workflow,*Workflow*>> executions.
+
+Service subscriptions are unique for each service. They can be enabled or disabled for each particular service.
+
+Job Subscriptions can be multiple, one can create various subscriptions for various kind of workflow types.
+
+By default each user has a Scheduler and a Job Planner subscription.
+To start receiving notifications, users only needs to choose notification methods,
+event types and activate the <<_glossary_notification_subscription,*Subscription*>>.
+
+On the other hand, Job subscriptions must be defined and created to receive Job notifications.
+To create a job subscription, go to the <<_glossary_notification_portal,*Notification Portal*>>,
+job subscriptions tab and click on the _+_ button
+
+image::job-subscription-creation-button.png[align="center"]
+
+Then fill and select the mandatory fields and click on _Save_
+
+image::job-subscription-creation.png[align="center"]
+
+At least one field of <<_glossary_workflow,*Workflow*>> data must be provided to create the subscription, that means it
+is not possible to create a job subscription that will listen for all jobs submitted to the Scheduler.
+
+Two scenarios are possible:
+
+- If a bucket name is only set, then the user will be notified for all <<_glossary_workflow,*Workflows*>> of the bucket
+- Otherwise, the bucket name, project name and workflow name of the workflow you wish to be notified of
+must exactly match with the bucket name, project name and workflow name of the subscription.
+
+The process is the same for Scheduler and Job_planner subscriptions without the
+need to provide <<_glossary_workflow,*Workflow*>> data. Select notification methods,
+events and create the <<_glossary_notification_subscription,*Subscription*>>.
+
+To modify a subscription simply click on it and do your adjustments.
+
+==== Notification methods
+
+To define the way to be notified, the user must activate at least one notification method.
+It is mandatory to activate at least one to be able to activate a subscription.
+
+There are three notification methods available
+
+- "Portal": The notification will be displayed in the <<_glossary_notification_portal,*Notification Portal*>>.
+
+- "Email": The notification will be sent by email using SMTP protocol hence SMTP server related data
```

Comment:
using the SMTP

Reply:
I order to use this method, the SMTP settings must be defined in the dedicated configuration file, inside the ProActive server.

Please refer to the corresponding link:../admin/ProActiveAdminGuide.html#_email_notification_method[Email notification configuration] section in the administration guide.